src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/activities/AttendanceResourceIntTest.kt

```diff
@@ -171,10 +171,10 @@ class AttendanceResourceIntTest : IntegrationTestBase() {
 
       @Test
       fun `should return bad request if offender booking not found`() {
-        webTestClient.upsertAttendance(courseSchedule.courseScheduleId, 222)
+        webTestClient.upsertAttendance(courseSchedule.courseScheduleId, 22222)
           .expectStatus().isBadRequest
           .expectBody().jsonPath("userMessage").value<String> {
-            assertThat(it).contains("Offender booking with id=222 not found")
+            assertThat(it).contains("Offender booking with id=22222 not found")
```

Comment:
Fix 3 - this test started randomly failing - suspect there are more than 221 offender bookings created by the test suite.